docs/advanced-features/middleware.md

```diff
@@ -152,6 +152,24 @@ export function middleware(request: NextRequest) {
 }
 ```
 
+## Web Assembly (Wasm)
+
+You can use Wasm within your Middleware by importing your `.wasm` binary with:
+
+```ts
+import wasm from './my-file.wasm?module'
+```
+
+Note the `?module` suffix. This will provide an array of the WASM data that can be instantiated using `WebAssembly.instantiate()`, which can be done as follows:
```

Comment:
```suggestion
Note the `?module` suffix. This will provide an array of the Wasm data that can be instantiated using `WebAssembly.instantiate()`, which can be done as follows:
```